react_components/PaginationBoxView.js

```diff
@@ -572,11 +572,7 @@ export default class PaginationBoxView extends Component {
     const nextAriaDisabled = isNextDisabled ? 'true' : 'false';
 
     return (
-      <ul
-        className={className || containerClassName}
-        role="navigation"
-        aria-label="Pagination"
-      >
+      <ul className={className || containerClassName} aria-label="Pagination">
```

Comment:
The hard-coded `aria-label` value of "Pagination" should ideally be removed as well, or if included it should be configurable (although I suppose that's a separate issue)

Reply:
@kroney, do you have a plan to adjust the comment?